packages/api/package.json

```diff
@@ -7,7 +7,8 @@
     "start": "nodemon",
     "build": "tsc",
     "lint": "eslint src --ext .js,.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "dev": "nodemon"
```

Comment:
This command shouldn't be necessary because it accomplishes the same thing as "start"